packages/react-components/react-motion/library/src/factories/createPresenceComponent.ts

```diff
@@ -1,4 +1,4 @@
-import { useEventCallback, useFirstMount, useIsomorphicLayoutEffect } from '@fluentui/react-utilities';
+import { type JSXElement, useEventCallback, useFirstMount, useIsomorphicLayoutEffect } from '@fluentui/react-utilities';
```

Comment:
this should be separate `import type {}` seems our ts-minbar doesn't account for this library ups ( otherwise CI would fail )